### PR TITLE
BE - Add long/lat and address to MonitoringLocation

### PR DESCRIPTION
**Pull request by BenjaminJ**
## Jira link(s)
- https://issues.opennms.org/browse/HS-1539

## Checklist
* [ ] Follows Horizon Stream's [development guidelines.](https://github.com/OpenNMS/horizon-stream/wiki/Development-Guidelines)
* [ ] Appropriate reviewer(s) have been selected.
* [ ] Jira issue(s) have been updated to "In Review".
* [ ] Includes [appropriate tests.](https://github.com/OpenNMS/horizon-stream/wiki/Test-Strategy)
* [ ] Documentation has been updated as necessary.
* [ ] Notify devops of changes to the Charts

### CODE REVIEW

**Review by splatch — APPROVED** [review]

File: inventory/main/src/main/resources/db/changelog/hs-0.1.0/tables/monitoring-location.xml [hunk 1]
```diff
@@ -34,4 +34,22 @@
         />
 
     </changeSet>
+
+    <changeSet author="bjanssens" id="0.1.0-add-geo-monitoring-location-columns">
+        <addColumn tableName="monitoring_location">
+            <column name="latitude" type="DOUBLE">
```
Comment (cgorantla) [inline inventory/main/src/main/resources/db/changelog/hs-0.1.0/tables/monitoring-location.xml:40]: Is it worth having latitude/langitude as double ?  May be strings are sufficient as these are mostly constants.

File: inventory/main/src/main/java/org/opennms/horizon/inventory/model/MonitoringLocation.java [hunk 2]
```diff
@@ -56,4 +54,15 @@ public class MonitoringLocation {
...
+    @NotNull
+    @Column(name = "longitude")
+    private double longitude;
+
+    @NotNull
+    @Column(name = "latitude")
+    private double latitude;
+
+    @Column(name = "address")
+    private String address;
```
Comment (cgorantla) [inline inventory/main/src/main/java/org/opennms/horizon/inventory/model/MonitoringLocation.java:67]: All of them should be optional.  Once we move longitude/latitude to be strings, these don't be not null

**Review by cgorantla — CHANGES_REQUESTED** [review]

File: inventory/main/src/main/java/org/opennms/horizon/inventory/model/MonitoringLocation.java (hunk 2 above)
Comment (freddy61025) [reply inventory/main/src/main/java/org/opennms/horizon/inventory/model/MonitoringLocation.java:67]: I will add a proper fix in https://opennms.atlassian.net/browse/HS-1648

File: inventory/main/src/main/resources/db/changelog/hs-0.1.0/tables/monitoring-location.xml (hunk 1 above)
Comment (freddy61025) [reply inventory/main/src/main/resources/db/changelog/hs-0.1.0/tables/monitoring-location.xml:40]: I think double can save more space and easier for searching.

**Pull request merged**